src/BlackSlope.Api.Common/Versioning/Services/JsonVersionService.cs

```diff
@@ -18,10 +18,9 @@ public JsonVersionService(IFileSystem fileSystem, IWebHostEnvironment hostingEnv
 
         public Version GetVersion()
         {
-            var filepath = _fileSystem.Path.Combine(_hostingEnvironment.ContentRootPath, "Common", "Version", "version.json");
+            var filepath = _fileSystem.Path.Combine(_hostingEnvironment.ContentRootPath, "..", "Blackslope.Api.Common", "Versioning", "version.json");
```

Comment:
Please create a new issue to fix this. The plan is to make 'Common' a reusable nuget package, this approach does not work.

Reply:
Done.

https://github.com/SlalomBuild/blackslope.net/issues/35